### PR TITLE
Quiz App

### PR DESCRIPTION
**Pull request by Programing-School**

### CODE REVIEW

**Comment by Programing-School** [comment]
Please A Feedback on the README

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ What all can you make:
 - Hero section template
 - Navigation Bar
 - Projects Section
+- Quiz App
```
Comment (mudit023) [inline README.md:15]: Don't update this everytime you add a new template

File: README.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ What all can you make:
...
+- Gallery
```
Comment (mudit023) [inline README.md:16]: revert these changes

**Review by mudit023 — COMMENTED** [review]
Go through the review and make necessary chnages.

File: README.md (hunk 1 above)
Comment (Programing-School) [reply README.md:15]: Ok, I am sorry

File: README.md (hunk 2 above)
Comment (Programing-School) [reply README.md:16]: Ok

**Comment by Programing-School** [comment]
Success

**Comment by Programing-School** [comment]
please slack workspace url

**Comment by mudit023** [comment]
There is some problem with this PR. Can you make a new one?
